Morpheus "Apollo" gentx preparation

rmontagnin opened the pull request:
This PR prepares the final `genesis.json` file for the upcoming `morpheus-apollo-1` testnet. The following changes have been applied: 

1. Updated the `x/auth` accounts list, to include all the genesis validators' delegators. 
2. Updated the `x/bank` balances list, to give each of the genesis validators' delegator `10000000udaric` (10 Daric). 
3. Run the `desmos collect-gentxs` command to collect all the valid genesis transactions. 
4. Increased `max_validators` to `150`
5. Added vesting account

Closes #5, closes #43, closes #201, closes #274

## Note on the genesis transactions
There have been a total of 193 submitted genesis transactions. Of all of these, I deleted

- all the invalid ones (empty body, or created with an old `desmos` version)
- all the duplicate ones (either same validator, same delegator or both)
- all the ones that came from people that did not submit the validators' survey in time
- all the ones that came from people that submit wrong values inside the validators' survey (eg. wrong operator address)

rmontagnin committed:
Added genesis accounts and balances

rmontagnin committed:
Removed duplicate genesis transactions

rmontagnin committed:
Removed duplicate account

rmontagnin committed:
Updated genesis time

rmontagnin committed:
Updated genesis.json

rmontagnin committed:
Collected genesis transactions

rmontagnin committed:
Updated README with correct SHA-256 hash and genesis time

leobragaz commented on morpheus-apollo-1/genesis.json:
```diff
@@ -30,51 +30,11142 @@
+          "account_number": "0",
+          "sequence": "0"
+        },
+        {
```
remove from here to 481

rmontagnin replied:
Done

leobragaz commented on morpheus-apollo-1/genesis.json:
```diff
@@ -30,51 +30,11142 @@
+                "description": {
+                  "details": "test-drill",
+                  "identity": "",
+                  "moniker": "leobragaz",
```
remove this validator

rmontagnin replied:
Done

kwunyeung reviewed (changes requested):
I can successfully start a node if I move the genesis time to past time. The current 144 accounts and genesis txs are correct.

Let's update the `max_validators` to `150`.

kwunyeung commented on morpheus-apollo-1/genesis.json:
```diff
@@ -30,51 +30,11142 @@
+              "amount": "10000000"
+            }
+          ]
+        }
```
Add bank balances for the vesting account
```suggestion
        },
        {
          "address": "desmos1snmetx9sfrmznr4z6d3lnwlyvwu36sgcpzzhlr",
          "coins": [
            {
              "denom": "udaric",
              "amount": "300000000000"
            }
          ]
        }
```

kwunyeung commented on morpheus-apollo-1/genesis.json:
```diff
@@ -30,51 +30,11142 @@
+          "pub_key": null,
+          "account_number": "0",
+          "sequence": "0"
+        }
```
Add a new vesting account with vesting period
```suggestion
        },
        {
          "@type": "/cosmos.vesting.v1beta1.PeriodicVestingAccount",
          "base_vesting_account": {
            "base_account": {
              "address": "desmos1snmetx9sfrmznr4z6d3lnwlyvwu36sgcpzzhlr",
              "pub_key": null,
              "account_number": "0",
              "sequence": "0"
            },
            "original_vesting": [
              {
                "denom": "udaric",
                "amount": "30000000000"
              }
            ],
            "delegated_free": [],
            "delegated_vesting": [],
            "end_time": "1619766000"
          },
          "start_time": "1619506800",
          "vesting_periods": [
            {
              "length": "86400",
              "amount": [
                {
                  "denom": "udaric",
                  "amount": "10000000000"
                }
              ]
            },
            {
              "length": "86400",
              "amount": [
                {
                  "denom": "udaric",
                  "amount": "10000000000"
                }
              ]
            },
            {
              "length": "86400",
              "amount": [
                {
                  "denom": "udaric",
                  "amount": "10000000000"
                }
              ]
            }
          ]
        }
```

rmontagnin committed:
Add a new vesting account with vesting period

Co-authored-by: Kwun Yeung <kwun@forbole.com>

rmontagnin committed:
Add bank balances for the vesting account

Co-authored-by: Kwun Yeung <kwun@forbole.com>

rmontagnin committed:
Removed leobragaz test validator

rmontagnin committed:
Increased max validators to 150

rmontagnin committed:
Changes
- Updated genesis transactions submissions with missing validators
- Removed gentxs from genesis

rmontagnin committed:
Added missing validators gentxs

rmontagnin committed:
Added missing form entry

rmontagnin committed:
Collected gentxs and updated README

rmontagnin committed:
Removed balance not correlated with any account

rmontagnin committed:
Updated genesis SHA-256

rmontagnin committed:
Added faucet account

rmontagnin committed:
Updated genesis hashsum

leobragaz reviewed (approved):
🚀
